### DISCUSSION TITLE
When selecting multiple Books for editing, allow to "Add tags"

### DISCUSSION THREAD

**Discussion by Git08hb** [mateusz-bak/openreads#741] [Ideas]
So i have rad lots of comic books and just found out I could use tags to have a better and clear view when I am looking for an specific comic book series. My feature request is to be able to select multiple books and then add the same tags to them at the same time.

Thank you!

**Comment by mateusz-bak** [comment]
This is part of https://github.com/mateusz-bak/openreads/discussions/787
